src/main/include/LimelightUtil.h

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc2/command/SubsystemBase.h>
+#include <frc2/command/CommandPtr.h>
+#include <iostream>
+#include <vector>
+#include <units/length.h>
+//#include <units/angle.h>
+#include <cmath>
+
+#include "LimelightHelpers.h"
+
+namespace t34
+{
+
+  class LimelightUtil : public frc2::SubsystemBase 
+  {
+  public:
+
+    LimelightUtil(std::string nt_name, units::degree_t camera_angle);
+
+    void Periodic() override;
+    
+    units::inch_t CalcDistance();
```

Comment:
Please provide a comment here. Something like the following:

```suggestion
    //! \brief Calculates the distance to I'm-not-sure-what.
    //! \returns The distance to I'm-not-sure-what.
    units::inch_t CalcDistance();
```